MovieAppPetProject/MovieAppPetProject/HomeScreen/NewHomeViewController.swift

```diff
@@ -77,13 +78,21 @@ class NewHomeViewController: UIViewController, UITableViewDelegate, UITableViewD
             cell.configure(with: upcomingMoviesViewModel.upcomingMovies ?? [], sectionTitle: " Upcoming")
             cell.didSelectItem = { [weak self] movie in
                 guard let self = self else { return }
-                self.performSegue(withIdentifier: Constants.Identifiers.goToMovieDetails, sender: movie)
+                self.performSegue(withIdentifier: Constants.Identifiers.goToMovieDetails, sender: movie.movieID)
             }
             return cell
         default:
             return UITableViewCell()
         }
     }
+
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.identifier == "GoToMovieDetails",
```

Comment:
Segue identifiers can be moved to constants file

Reply:
thank you